test/acceptance/features/immutableBindings.feature

```diff
@@ -0,0 +1,192 @@
+Feature: Successful Service Binding are Immutable
+
+    As a user of Service Binding operator
+    I should not be able to modify a Service Binding that is ready
+
+    Background:
+        Given Namespace [TEST_NAMESPACE] is used
+        * Service Binding Operator is running
+        * CustomResourceDefinition backends.stable.example.com is available
+        * The Custom Resource is present
+        """
+        apiVersion: stable.example.com/v1
+        kind: Backend
+        metadata:
+            name: service-immutable
+            annotations:
+                service.binding/host: path={.spec.host}
+        spec:
+            host: foo
+        """
+
+    Scenario: Cannot update a Service Binding that is ready
+        And Generic test application "app-immutable" is running
+        And Service Binding is applied
+            """
+            apiVersion: binding.operators.coreos.com/v1alpha1
+            kind: ServiceBinding
+            metadata:
+                name: binding-immutable
+            spec:
+                services:
+                  - group: stable.example.com
+                    version: v1
+                    kind: Backend
+                    name: service-immutable
+                application:
+                    name: app-immutable
+                    group: apps
+                    version: v1
+                    resource: deployments
+            """
+        When Service Binding "binding-immutable" is ready
+        Then Immutable Service Binding is unable to be applied
```

Comment:
IMHO, the word "Immutable" is unnecessary here, in the step name. Every binding that is ready is supposed to be immutable, so I suggest to name it just `Service Binding is unable to be applied"